src/routes/tickets/controllers/getUnassignedTickets.ts

```diff
@@ -23,7 +23,7 @@ const getUnassignedTickets = async (req: Request, res: Response) => {
   const tickets: IPaginateResult<ITicket> = await find(getDB().tickets, {
     limit: parseInt(process.env.PAGINATION_LIMIT, 10),
     query: {
-      status: TicketStatus.updated,
+      status: TicketStatus.open,
```

Comment:
Is this meant to be open? I noticed you said you would revert this elsewhere

Reply:
Fixed.